templates/threatconnect.hbs

```diff
@@ -1,22 +1,59 @@
 {{#if block.detailsLoaded}}
   {{#each results as |orgData orgDataIndex|}}
     {{#if (gt orgDataIndex 0)}}
-      <hr>
+      <hr />
     {{/if}}
 
     <ul class="nav nav-tabs mb-2">
       <li class="nav-item">
-        <a {{action "changeTab" "info" orgDataIndex}} class="nav-link {{if (or (eq orgData.__activeTab "info")(not orgData.__activeTab)) "active"}}" href="#">Info</a>
+        <a
+          {{action "changeTab" "info" orgDataIndex}}
+          class="nav-link {{if (or (eq orgData.__activeTab "info") (not orgData.__activeTab)) "active"}}"
+          href="#"
+        >Info</a>
       </li>
       <li class="nav-item">
-        <a {{action "changeTab" "associations" orgDataIndex}} class="nav-link {{if (eq orgData.__activeTab "associations") "active"}}" href="#">Associations ({{orgData.numAssociations}})</a>
+        <a
+          {{action "changeTab" "associations" orgDataIndex}}
+          class="nav-link {{if (eq orgData.__activeTab "associations") "active"}}"
+          href="#"
+        >Associations ({{orgData.numAssociations}})</a>
       </li>
+      {{#if orgData.dnsInformation.result}}
+        <li class="nav-item">
+          <a
+            {{action "changeTab" "dnsInformation" orgDataIndex}}
+            class="nav-link {{if (eq orgData.__activeTab "dnsInformation") "active"}}"
+            href="#"
+          >DNS Resolutions</a>
+        </li>
+      {{/if}}
     </ul>
 
+    {{#if (eq orgData.__activeTab "dnsInformation")}}
+      <div>
+        <span>{{fa-icon icon="network-wired"}}</span>
+        <span>DNS Resolutions</span>
+      </div>
+      {{#each orgData.dnsInformation.result as |info|}}
+        <div class="dns-information">
+          {{#each info.addresses as |addressField|}}
+            {{#each-in addressField as |key value|}}

```

Comment:
So sadly this solution wont work from a UI/UX standpoint with this object.  This solution only works with objects that just contain booleans, strings, and numbers.  With the DNS objects we get dates, a link, and an ID we don't want to display, so sadly we will **_need to add specific key value pairs here with the additional formatting._**
<img width="200" alt="Screen Shot 2021-09-01 at 6 03 49 PM" src="https://user-images.githubusercontent.com/58269952/131756672-bcfa4bc5-1745-4e7f-b949-24bcee9ed5c9.png">

However, if in the future you do find an object you want to display with just booleans, strings, and numbers, you will need to use some string helpers on the key so the key would look like `{{titleize (humanize (dasherize key))}}`.  This turns ![Screen Shot 2021-09-01 at 6 38 14 PM](https://user-images.githubusercontent.com/58269952/131759059-1af4fd4e-1467-450b-9164-9d98662d3ac6.png) into ![Screen Shot 2021-09-01 at 6 38 31 PM](https://user-images.githubusercontent.com/58269952/131759076-d84a0e6e-2305-4b5f-b3dc-0ddd9f8bc269.png).

Reply:
So the updates you made solve a few of the issues I mentioned above but a few still remain.  
- [ ] We don't want to display the Id so _**the Id needs removed from the displayed results**_.  
- [ ] We need a specific order to the fields as having the link and ip/domain at the bottom looks a bit out of order.  Usually the order in this case would be link, ip, owner, threat/confidence, then date info. _**The order of the fields should be changed so the most relevant info is at the top**_.
- [ ] Our links don't usually have a key, and we usually say "View in __platform".  _**The key will need to be removed from the link**_
- [ ] The `Last Modified` key value isn't formatted.  _**The Last Modified value needs to use the moment-format ember helper**_

However these issues are solved is fine, but I would recommend switching over explicitly labeling the key value pairs with conditional rendering for each key, as the ordering issues and excluding the id would likely most easily be solved in this format, and be clearest from a readability standpoint down the road.

<img width="281" alt="Screen Shot 2021-09-07 at 11 03 29 AM" src="https://user-images.githubusercontent.com/58269952/132376314-e0b0a90e-bafd-46a0-b3e7-d650ae3519a3.png">